src/main/fc/settings.yaml

```diff
@@ -3211,6 +3210,18 @@ groups:
         field: sidebar_scroll_arrows
         type: bool
         default_value: OFF
+      - name: osd_adsb_range
+        description: "Distance outside which adsb aircraft are NOT displayed. In meters"
+        default_value: 20000
+        field: adsb_range
+        min: 0
```

Comment:
Default min is 0 so this is not needed but it won't hurt

Reply:
noted. done